libraries/abstractions/secure_sockets/test/iot_test_tcp.c

```diff
@@ -1554,8 +1555,22 @@ static void prvSOCKETS_NonBlocking_Test( Server_t xConn )
                  */
                 xNumBytesReceived += lNumBytes;
             }
+            else if( ( lNumBytes < 0 ) && ( lNumBytes != SOCKETS_EWOULDBLOCK ) )
+            {
+                /* Since we are in non blocking mode, this error is expected if the TCP stack is
```

Comment:
expected or not expected?

Reply:
Yes, this error is expected since this socket is in non-blocking mode.